packaging/generic/build-image/inside/docker-entrypoint.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+
+scl enable devtoolset-3 -- "$@"
```

Comment:
Don't think the quotes on `$@` are necessary.

Reply:
They are there for the centos6 package and shellcheck complains without those :-P